docs/getting-started/advanced-installation.md

```diff
@@ -17,13 +17,30 @@ If you have problem with downloading, use this:
 Invoke-WebRequest -UseBasicParsing "https://raw.githubusercontent.com/spicetify/spicetify-cli/master/install_curl.ps1" | Invoke-Expression
 ```
 
+### Chocolatey
+
+Follow this guide: https://chocolatey.org/packages/spicetify-cli
+
 ### Scoop
 
-Follow this guide: https://github.com/zwxi/Scoop-Spotify
+```powershell
+scoop install spicetify-cli
+```
 
-### Chocolatey
+### Note for Windows users
 
-Follow this guide: https://chocolatey.org/packages/spicetify-cli
+#### Spotify installed from Scoop
+
+- To find the location of your Spotify installation, run `scoop prefix spotify`.
+
+```console
+$ scoop prefix spotify
+C:\Users\<username>\scoop\apps\spotify\current
+```
+
+After you have located it, set `spotify_path` to that directory in the config file:
+
+![scoop-spotify-path](https://user-images.githubusercontent.com/56180050/157407968-9d55a75a-bcd5-49ec-bbd1-5f315bd3c303.png)
```

Comment:
I would remove this as it does not add much information, and the line numbering is somewhat confusing 😕

Reply:
I was being consistent with the Flatpak Linux installation, but I guess I can fix the line numbering.

Reply:
What is already there isn't the best example 😅 Thanks!